### ISSUE TITLE
HandsontableCopyPaste textarea causing "Missing form label" accessibility error

### ISSUE THREAD

**Issue by joelamos** [handsontable/handsontable#5786]
### Description
The `textarea` with id="HandsontableCopyPaste" is causing a "Missing form label" when evaluated by the WAVE tool.

### Steps to reproduce
1. Install the [WAVE browser extension](https://wave.webaim.org/extension/), a "web accessibility evaluation tool."
2. Navigate to [this](https://handsontable.com/examples?sorting) example table.
3. Run WAVE against the page.

### Demo
![image](https://user-images.githubusercontent.com/5117047/52646296-857a0900-2eb0-11e9-9cd3-128ae5580c79.png)

### Your environment
* Handsontable version: 6.2.2
* Browser Name and version: 71.0
* Operating System: Windows 10

**Comment by AMBudnik** [comment]
Hey @joelamos 

thank you for sharing. I think that we should add the ARIA support first https://github.com/handsontable/handsontable/issues/5596

**Comment by joelamos** [comment]
@AMBudnik Makes sense. In any case, do you have a recommendation for what a suitable value for `aria-label` (or other attribute) could be? This seems to be a backup element that gets focused in the case of a missing editor on copy/paste/cut with the sole purpose of capuring those events.

**Comment by AMBudnik** [comment]
I'm not an expert but I asked one of our developers to check the subject.

**Comment by AMBudnik** [comment]
I got a reply

> aria-label="Listens for copy, cut, and paste events." I think user is right

**Comment by joelamos** [comment]
Great, thanks!

**Comment by joelamos** [comment]
Hey @AMBudnik, this is great news per accessibility conformance! I'll relay the info to my company.
